src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -85,10 +91,96 @@ public void after() throws SQLException {
     log.info("Dropped table");
   }
 
+  /**
+   * Verifies that even when the connector encounters exceptions that would cause a connection
+   * with an invalid transaction, the connector sends only the errant record to the error
+   * reporter and establishes a valid transaction for subsequent correct records to be sent to
+   * the actual database.
+   *
+   * @throws Exception
```

Comment:
(Not a blocker) Never seen a `@throws` Javadocs annotation for a test before, tbh it seems better to leave this out if it doesn't provide any useful information to people looking at the test code since it's not like this method is going to be called by external classes where the developer might want to know "hmmm, what checked exceptions are thrown from this method and why?"